src/acceleration/BaseQNAcceleration.hpp

```diff
@@ -287,6 +289,17 @@ class BaseQNAcceleration : public Acceleration {
    */
   virtual void specializedInitializeVectorsAndPreconditioner(const DataMap &cplData) = 0;
 
+  /// @brief Samples and concatenates the data and old data in cplData into a long vector
+  void concatenateCouplingDataWaveform(Eigen::VectorXd &data, Eigen::VectorXd &oldData, const DataMap &cplData, std::vector<int> dataIDs, impl::WaveformTimeGrids timeGrids) const;
```

Comment:
This was done to differentiate between the normal concatenateCouplingData that lives the Aitken acceleration, but I can remove the addition to the name since they are in two different files.